iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -21,6 +21,8 @@
 #include "iceoryx_posh/iceoryx_posh_types.hpp"
```

Comment:
Please adapt the copyright header.

Reply:
It is already correct. It seems some merge switched the order. I'll change it back to have the creator of the file as first entry